site/public/css/electronic.css

```diff
@@ -236,16 +236,16 @@ progress::-webkit-progress-value {
     color: var(--text-black);
     position: relative;
     cursor: pointer;
-    padding: 2px 6px 5px 24px;
+    padding: 4px 12px 5px 25px;
```

Comment:
While this does technically center the text in the button, I think it would be better to do it without changing the size of the button.  Thicker buttons mean more screen real estate is lost.

I'm interested in other opinions on this though.  Does anyone have thoughts on this?

Reply:
Its looks good in the site that's why i made these changes if you want to change it i can change it without increasing the size of the button.

Reply:
@KUNDAN1334 please make the changes william requested and then we can go from there, thank you!

Reply:
@micpap25 sure